lib/vast/Frontend/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ add_vast_library(Frontend
     Targets.cpp
 
     LINK_LIBS PUBLIC
+    gap::gap-sarif
```

Comment:
Is there a way to make this an optional dependency?

Reply:
I guess this could be gated behind a CMake option

Reply:
Ideally, yes please!

Reply:
Will need https://github.com/lifting-bits/gap/pull/53